docs/faq/asusctl.md

```
@@ -0,0 +1,54 @@
+### I don't have any power profiles or charge control
+
+We recommend to use at least 6.19 so that you get all the most recent patches and fixes for ASUS laptops.
+
+It's also possible that your laptop doesn't support this so if the kernel update doesn't solve this feel free to make a :sadface: (sorry).
+
+### How do I set a custom fan curve?
+
+Custom fan curves (not speaking of the built in power profiles) are currently only supported on Ryzen ROG laptops.
+The necessary kernel patches are merged since 5.17.
+
+The format is shown [here](https://github.com/cronosun/atrofac/blob/master/ADVANCED.md#limits).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)